sys/kern/vfs_cache.c

```diff
@@ -190,7 +194,10 @@ TAILQ_HEAD(cache_freebatch, namecache);
  * tied to VFS SMR. Even if retaining them, the current split should be
  * re-evaluated.
  */
-#ifdef __LP64__
+#ifdef __CHERI_PURE_CAPABILITY__
+#define	CACHE_PATH_CUTOFF	45
+#define	CACHE_LARGE_PAD		14
```

Comment:
So this is just `sizeof(void *) - 2`... bleh crappy upstream code

Reply:
Yeah..